pyccl/pk2d.py

```diff
@@ -229,13 +283,26 @@ def apply_halofit(self, cosmo, pk_linear=None):
         return pk2d
 
     def eval(self, k, a, cosmo=None, *, derivative=False):
-        """Evaluate the power spectrum or its logarithmic derivative.
+        warnings.warn("Pk2D.eval is deprecated. Simply use the object's "
+                      "__call__ method.", category=CCLDeprecationWarning)
```

Comment:
I would phrase it as "simply call the instance" or "simply call the object" because users might not know that `__call__` can be used directly like this.

Reply:
Done